.github/workflows/danger.yml

```diff
@@ -0,0 +1,28 @@
+name: Danger
+
+on: pull_request
+
+jobs:
+  danger:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7.2
+        bundler-cache: true
+    - uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('Gemfile') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
+    - uses: MeilCli/danger-action@v5
+      with:
+        plugins_file: 'Gemfile'
+        install_path: 'vendor/bundle'
+        danger_file: 'Dangerfile'
+        danger_id: 'danger-pr'
+      env:
+        DANGER_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think this will work, I had to do something like https://github.com/dblock/strava-ruby-client/blob/0d7f31cf1f839d9a7a727832e182d8fb8c47d6cc/.github/workflows/danger.yml

Reply:
Thanks, turned out it only worked in my fork but not in this main repository. I've just updated with your example. Can you please have a look [here](https://github.com/ashkan18/graphlient/pull/91)?